tools/restaked/src/main.rs

```diff
@@ -22,7 +22,7 @@ fn maybe_kicked_out(validator_info: &CurrentEpochValidatorInfo) -> bool {
 
 fn main() {
     let filter = near_o11y::EnvFilterBuilder::from_env().verbose(Some("")).finish().unwrap();
-    let _subscriber = near_o11y::default_subscriber(filter, ColorOutput::Auto).global();
+    let _subscriber = near_o11y::default_subscriber(filter, &ColorOutput::Auto).global();
```

Comment:
I believe these might just panic since there isn't a runtime set-up for `default_subscriber` to use.

Reply:
Fixed